client/css/style.css

```diff
@@ -1672,6 +1696,17 @@ kbd {
 	border-right-color: #222;
 }
 
+@media only screen and (-webkit-min-device-pixel-ratio: 2),
```

Comment:
This comes from [here](https://github.com/primer/primer-tooltips/blob/a8b5f34d4f5e0d019005d4c97610e19a0e528152/lib/tooltips.scss#L276-L280). I'd keep it as long as it's in the upstream project.